server/models/Customer.js

```diff
@@ -166,67 +166,65 @@ class Customer extends Model {
     }
   }
 
-  // Update the payment source for the customer (and make it the default)
-  async updateSource(source) {
+  async updatePaymentMethod(paymentMethodId) {
     try {
-      // Stripe API: Attach the new source, update the default source
-      const createdSource = await stripe.customers.createSource(this.stripeId, {
-        source: source.id,
-      });
-      const updatedCustomer = await stripe.customers.update(this.stripeId, {
-        default_source: createdSource.id,
-      });
+      // Attach PaymentMethod to Customer
+      const { card } = await stripe.paymentMethods.attach(
+        paymentMethodId,
+        { customer: this.stripeId }
+      );
 
-      // DB: Update the payment source ID
+      // DB: Update the payment method ID
       const updated = await db(this.constructor.table)
         .where('id', this.id)
         .update({
-          sourceId: createdSource.id,
-          sourceLast4: createdSource.last4,
-          sourceBrand: createdSource.brand,
+          paymentMethodId: paymentMethodId,
+          paymentMethodLast4: card.last4,
+          paymentMethodBrand: card.brand,
         });
 
-      // Update our subscription to charge automatically: we've got a source
+      // Fetch Subscription
       const subscription = await this.getSubscription();
-      const paymentMethod = 'charge_automatically';
-      // Update the payment method if it changed
-      if (subscription && subscription.billing != paymentMethod) {
-        // Update the payment method if it changed
+      
+      const collectionMethod = 'charge_automatically';
+
+      if (subscription && subscription.collectionMethod != collectionMethod) {
+        // Make PaymentMethod the default for Subscription
         const updatedSubscription = await stripe.subscriptions.update(
           subscription.stripeId,
-          {billing: paymentMethod}
+          {
+            default_payment_method: paymentMethodId,
+            collection_method: collectionMethod,
+          }
         );
+
         const updatedBilling = await db(Subscription.table)
           .where('id', subscription.id)
-          .update({billing: paymentMethod});
+          .update({collectionMethod: collectionMethod});
```

Comment:
With this changing billing to collectionMethod, we need to update collectionMethod for invoice settings on line 251 https://github.com/stripe/stripe-billing-typographic/pull/25/files#diff-7f8dc7dd9c244818d5292c60b3dfcfddR251